podman-desktop

```diff
@@ -18,8 +18,52 @@ die() {
   exit 1
 }
 
-command -v multipass >/dev/null 2>&1 || die "missing multipass"
-command -v podman >/dev/null 2>&1 || die "missing podman"
+brew() {
+  set +e
+  if [ -z "$(command -v brew)" ]; then
+    echo "# Install homebrew for easier MacOS management"
+    # shellcheck disable=SC2016
+    echo '/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"'
+  fi
+  set -e
+  echo "brew install $1"
+}
+
+validate() {
+  cmd=$1
+  if [ ! -x "$(command -v "$cmd")" ]; then
+    echo "ERROR: \`$cmd\` command not found. Please install $cmd." >&2
+    echo "Instructions are:"
+    if [ "$GOOS" = "linux" ]; then
+      echo "apt-get install $cmd"
+    elif [ "$GOOS" = "darwin" ]; then
+      brew "$cmd"
```

Comment:
Not a fan of installing things for the user

Reply:
Actually, this one should call the brew function above. I'll rename the function to make it more unique.